engine/gamesys/src/gamesys/scripts/script_collectionproxy.cpp

```diff
@@ -93,6 +95,7 @@ namespace dmGameSystem
         params.m_UrlHash = url_hash;
         params.m_OnlyMissing = only_missing;
         params.m_Recursive = false;
+        params.m_IncludeRequestedUrl = true;
```

Comment:
Since we take dependencies for collection now, the collection itself will not be included. This option includes the requested url itself into the result to keep functionality as it is now.